Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
I prefer not to add comments to the code, but if necessary I will do it...

First I will try to make the code more self explanatory.
My first intention was not to change a lot of code in that function because otherwise it would be harder to review but it's true that the code is a bit hard to understand.